lib/chef/provisioning/aws_driver/driver.rb

```diff
@@ -263,6 +265,26 @@ def destroy_image(action_handler, image_spec, image_options)
       end
     end
 
+    def user_data
+      # TODO: Make this use HTTPS at some point.
+      <<EOD
+<powershell>
+winrm quickconfig -q
+winrm set winrm/config/winrs '@{MaxMemoryPerShellMB="300"}'
+winrm set winrm/config '@{MaxTimeoutms="1800000"}'
+winrm set winrm/config/service '@{AllowUnencrypted="true"}'
+winrm set winrm/config/service/auth '@{Basic="true"}'
```

Comment:
I just want to be sure we're not ended up setting up instances in a way that's incompatible with knife-windows/winrm-s as part of our efforts there to standardize on more secure defaults. A while ago I ran into unexpected errors with these settings against development versions of those tools and I need to circle around on their state now.

https://github.com/chef/winrm-s/issues/13